src/Microsoft.PowerShell.Commands.Utility/commands/utility/TestJsonCommand.cs

```diff
@@ -239,32 +239,20 @@ protected override void ProcessRecord()
 
                 if (_jschema != null)
                 {
-                        var validationResults = _jschema.Validate(parsedJson, new ValidationOptions { OutputFormat = OutputFormat.Basic });
-                        result = validationResults.IsValid;
-                        if (!result)
-                        {
-                            if (validationResults.Message != null)
-                            {
-                                Exception exception = new(string.Format(TestJsonCmdletStrings.InvalidJsonAgainstSchemaDetailed, validationResults.Message, validationResults.InstanceLocation));
-                                ErrorRecord errorRecord = new(exception, "InvalidJsonAgainstSchemaDetailed", ErrorCategory.InvalidData, null);
-                                WriteError(errorRecord);
-                            }
+                    EvaluationResults evaluationResults = _jschema.Evaluate(parsedJson, new EvaluationOptions { OutputFormat = OutputFormat.List });
+                    result = evaluationResults.IsValid;
+                    if (!result)
+                    {
+                        HandleValidationErrors(evaluationResults);
 
-                            if (validationResults.HasNestedResults)
+                        if (evaluationResults.HasDetails)
```

Comment:
Previously, we had one top error and details errors. Now the design is changed. First of all, it is unexpected for a minor update. Secondly, it doesn't look as convenient. For example, if this cmdlet returns false, how would the user only get errors related to this check? 
@daxian-dbw Thoughts?

Reply:
No, it's the same thing. The property name just changed.  You still get all the same results. No functional change.

Reply:
Never mind.